### ISSUE TITLE
Setting the starting point

### ISSUE THREAD

**Issue by Julianoe** [mustardBees/cmb_field_map#39]
I wish i had time and knowledge to fork this to add a setting page or something like that to provide the possibility to define the starting point of the map (currently UK). For now i'll just modify the file ^^

**Comment by webdados** [comment]
+1

**Comment by gareth-gillman** [comment]
I didn't set a starting point but I did fork it to add the api keys, if you want I can update my fork to include the starting point.

https://github.com/gareth-gillman/cmb_field_map

**Comment by webdados** [comment]
@gareth-gillman Yes, you should please.

Because the starting point is on a JS file, I guess you should use wp_localize_script to pass the starting point coordinates from php to JS.

**Comment by gareth-gillman** [comment]
Yes I am aware as I do that with the api keys,  I will sort that this evening :)
